tests/io_test.py

```diff
@@ -243,14 +242,14 @@ def testGetDataSets(self):
         for dataset in datasets:
             self.assertIn(dataset, self.names)
 
-    @unittest.skip
     def testSaveStringsDataset(self):
         # Create, save, and load Strings dataset
-        strings_array = ak.array(['testing string{}'.format(num) for num in list(range(1,11))])
+        strings_array = ak.array(['testing string{}'.format(num) for num in list(range(0,25))])
         strings_array.save('{}/strings-test'.format(IOTest.io_test_dir), dataset='strings')
         r_strings_array = ak.load('{}/strings-test'.format(IOTest.io_test_dir), 
                                   dataset='strings')
-        self.assertTrue((strings_array == r_strings_array).all())
+        
+        self.assertTrue((strings_array.to_ndarray().sort() == r_strings_array.to_ndarray().sort()))
```

Comment:
I believe this should have a `.all()` on the end?

Reply:
Actually, `.sort()` is an in-place operation, so the return value is `None`, and this comparison will not function as intended. I think what you want is to call `np.sort(strings_array.to_ndarray())`, which will return a sorted copy of the array. Then you can compare with `(a == b).all()`

Reply:
@reuster986 Argh, yeah, fixed in latest version I just pushed to github.